JSDemos/Demos/DataGrid/GridSummaries/description.md

```diff
@@ -1 +1,11 @@
-The DataGrid component allows you to&nbsp;display data summaries within its container. This example illustrates the use of&nbsp;total summaries. The demo is&nbsp;configured using the **totalItems** array declared within the **summary** object. This array contains several objects that specify summary settings for each item.
\ No newline at end of file
+A total summary creates a synopsis for all data of a certain column. To configure this feature, add the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/) object, instatiate the **summary**.[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array and add summary objects. Each object must consist of at least the [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) and [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType) properties. The former specifies the column that supplies data, the latter sets the aggregate function that applies to this data.
```

Comment:
> To configure this feature, add the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/) object, instatiate the **summary**.[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array and add summary objects.

It's a difficult sentence to wrap one's head around because it gives instructions about code but the code itself is out there somewhere.

> instatiate 

means "to create an instance of something", which isn't what we do with the `totalItems` array.

> Each object must consist of

AFAIK, you can skip the `column` if you want a full data object in the `calculateCustomSummary` function.

```suggestion
A total summary creates a synopsis for all data of a certain column. To configure this feature, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
```